.github/ISSUE_TEMPLATE/new_podcast.yml

```diff
@@ -0,0 +1,71 @@
+name: New podcast
+description: Fill this form to add your podcast to the website.
+title: "[New][Podcast]: "
+labels: ["new", "podcast"]
+
+body:
+  - type: markdown
```

Comment:
I believe this one is not necessary (plus it does have a typo). The reason is because you already added a description input so it is pretty fine.

Reply:
I added a markdown field so we can just do whatever we want as a very good description of every type. So considerate as a placeholder for the preface of a type. We can remove it for this PR FYI